--- CMakeLists.txt.orig	2015-12-08 18:49:27.000000000 +0900
+++ CMakeLists.txt	2015-12-08 18:50:17.000000000 +0900
@@ -582,7 +582,7 @@
 configure_file( samples/common/include/CEGUISamplesConfig.h.in samples/common/include/CEGUISamplesConfig.h )
 configure_file( doc/doxygen/doxyfile.in doc/doxygen/doxyfile )
 
-if (UNIX AND NOT APPLE)
+if (UNIX)
     configure_file( cegui/CEGUI.pc.in cegui/CEGUI-${CEGUI_VERSION_MAJOR}.pc @ONLY )
     install(FILES ${CMAKE_BINARY_DIR}/cegui/CEGUI-${CEGUI_VERSION_MAJOR}.pc DESTINATION ${CEGUI_PKGCONFIG_INSTALL_DIR})
 
